hydra-plutus/src/Hydra/Contract/Head.hs

```diff
@@ -141,19 +128,19 @@ checkAbort ctx@ScriptContext{scriptContextTxInfo = txInfo} headCurrencySymbol pa
       commits
     TxInInfo{txInInfoResolved = txOut} : rest
       | hasPT headCurrencySymbol txOut ->
-        case commitDatum txInfo txOut of
-          Just commit ->
-            committedUTxO
-              (commit : commits)
-              rest
-          Nothing ->
-            committedUTxO
-              commits
-              rest
+          case commitDatum txInfo txOut of
+            Just commit ->
+              committedUTxO
+                (commit : commits)
+                rest
+            Nothing ->
+              committedUTxO
+                commits
+                rest
       | otherwise ->
-        committedUTxO
-          commits
-          rest
+          committedUTxO
+            commits
+            rest
```

Comment:
I would revert to the original code formatting so that we avoid future commit that will just reformat the old way because people are using the nix version of it.